docs/content/how-to/auth.md

```diff
@@ -0,0 +1,3 @@
+# How to use other authentication methods
+
+ToDo
```

Comment:
Instead of writing "ToDo" I would link an issue to let people know that this is intentionally left blank.